File: ExampleApp/app/src/main/java/com/motiv/example/MyApplication.java
package com.motiv.example;

import android.app.Application;
import io.realm.*;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
        RealmConfiguration config = new RealmConfiguration.Builder().build();
        Realm.setDefaultConfiguration(config);
    }
}

File: ExampleApp/app/src/main/java/com/motiv/example/MyApplication.kt
 
package com.motiv.example
import android.app.Application
import io.realm.*

public class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        val config = RealmConfiguration.Builder().build()
        Realm.setDefaultConfiguration(config)
    }
}
